index.html

```diff
@@ -40,4 +41,143 @@ <h1 class="heading">
+      <div class="project-card">
+
+        <img src="images/photo3.png" alt="Third Card">
+        <h2 class="head">Tonic</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+        <img src="images/photo4.png" alt="Forth Card">
+        <h2 class="head">Multi-post Stories</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+    </section>
+    <section class="section-about">
+        <div class="about-grid">
+          <div class="text">
+            <div class="headline-text">
+              <h1>About Myself</h1>
+
+              <p class="paragraph1">Hello I’m a software developer! I can help you build a product , feature or website
+                Look through some of my work and experience! If you like what you see and have a
+                project you need coded, don’t hestiate to contact me.</p>
+              <p class="lets-connect"> LET'S CONNECT</p>
+
+              <ul class="buttons getmyr">
+                <li><a href="#"> <img src="./images/twitter.png" alt="social"> </a></li>
+                <li><a href="#"> <img src="./images/Linkedin.png" alt="social"></a></li>
+                <li><a href="#"> <img src="./images/medium.png" alt="social"></a></li>
+                <li><a href="#"> <img src="./images/github.png" alt="social"></a></li>
+                <li><a href="#"> <img src="./images/angellist.png" alt="social"></a></li>
+              </ul>
+              <br>
+              <button class="btn-one btn-resume" type="button">Get My Resume</button>
+            </div>
```

Comment:
- [ ] This whole section has no space to the left which is not the case in the Figma design, It would be nice if you could rectify that.
- Your design:
![Capture8](https://user-images.githubusercontent.com/85113567/153602952-8824e197-bbe7-4bb4-9fc3-a42a4aaf1929.PNG)
- Figma design:
![Capture7](https://user-images.githubusercontent.com/85113567/153603001-740ce8da-3baa-43ee-aab1-612faec0ce00.PNG)

Reply:
👍🏽 All good now.